quarkus/tests/junit5/src/main/java/org/keycloak/it/junit5/extension/InfinispanContainer.java

```diff
@@ -51,10 +58,29 @@ public InfinispanContainer() {
             withImagePullPolicy(PullPolicy.alwaysPull());
         }
 
-
-        //order of waitingFor and withStartupTimeout matters as the latter sets the timeout for WaitStrategy set by waitingFor
-        waitingFor(Wait.forLogMessage(".*Infinispan Server.*started in.*", 1));
-        withStartupTimeout(Duration.ofMinutes(5));
+        // Waiting for the port to become available as waiting for the log was sometimes unreliable and seems to miss log messages at the beginning.
+        waitingFor(new AbstractWaitStrategy() {
```

Comment:
I tried to wait for a port binding and an open port, but that didn't work well as it conflicts with the `host` networking used here.